alerter-component/src/main/java/io/github/steliospaps/ighackathon/instrumentpricealerter/alertercomponent/AwsConfiguration.java

```diff
@@ -82,7 +72,7 @@ public AmazonCloudWatch amazonCloudWatch() {
 		private String region;
 
 		@Bean
-		public AWSCredentialsProvider amazonAWSCredentialsProvider() {
+		public AWSCredentialsProvider credentialsProvider() {
```

Comment:
is there a technical reason for this rename? (if we are over writting another bean we should be documenting this)

Reply:
Yes I renamed it as I overrode the spring cloud starter bean. I thought these beans may contain some special config but I found out they werent necessary. I have pushed another commit and deleted the bean definitions now (in favour of the spring cloud starter ones).